app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/SubscribedAndVerifierDetails.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cgtpropertydisposalsfrontend.models
+
+import cats.Eq
+import play.api.libs.json.{Format, Json}
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.EitherUtils.eitherFormat
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.address.Address
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.ids.CgtReference
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.name.{ContactName, IndividualName, TrustName}
+
+final case class SubscribedAndVerifierDetails(
```

Comment:
you probably could have got away with something like this:
```scala
final case class SubscriptionUpdateRequest(
  newDetails: SubscribedDetails
  previousDetails: SubscribedDetails
)
```
to avoid having all the same fields here again in this case class. We're going to have to have the old details when we come to doing the TxM audit events in the BE anyway, so we could make that work easier here.

Reply:
```scala
final case class SubscriptionUpdateRequest(
  newDetails: SubscribedDetails
  previousDetails: Option[SubscribedDetails]
)
```

The `previousDetails` would have to be optional.

Reply:
would the previous details not always be at hand?

Reply:
Ah, ok. I see what you mean. I was assuming we would only populate if there was an address change only. But yes if we generalise this then we could send both sets of data to the BE.

Reply:
yeah. then the BE can decide whether or not to update the verifiers based on the differences it sees between the old and new details. This way, if we ever change the verifiers (which we probably will never do) only the BE will have to change and the FE can be left alone

Reply:
Refactored the model + test. Need to do the the corresponding BE changes now.